src/Cake.Svn/Internal/SharpSvnClient.cs

```diff
@@ -113,7 +113,7 @@ public IEnumerable<SvnInfoResult> GetInfo(string repositoryUrl, SvnInfoSettings
                     select new SvnInfoResult(
                         eventArg.RepositoryId,
                         eventArg.RepositoryRoot,
-                        eventArg.LastChangeAuthor,
+                        eventArg.LastChangeAuthor ?? string.Empty,
```

Comment:
Why? What's the problem with returning `null`? Beside this change has nothing to do this the linked issue, so I'm confused why this should be changed.

Reply:
@pascalberger if we pass null to the ctor an Exception will be thrown as we check this parameter to null. LastChangeAuthor can be null if the file was added to the working copy but not yet committed. This case happens after change I did to nullable revision.

Reply:
@christianbumann Wouldn't it be better in this case to allow LastChangeAuthor to be null in `SvnInfoResult` ctor?

Reply:
@pascalberger yes this would be an option (I had this in the first commit of this PR) but changed to the string.Empty behaviour. If we change this to allow null, we should also allow null on FullPath...